### PR TITLE
More mutation sprites: Part 2

### PR DESCRIPTION
**Pull request by KittyTac**
#### Summary
Add sprites for Thick Scales, Antennae, and Tail Club.

#### Content of the change
See Summary. I took a bit of artistic license with Thick Scales: "Mutant Bone" covers 87% of arms and legs but I chose to draw it as rings of bone as that looked less weird.

#### Testing
изображение

#### Additional information
N/A

### CODE REVIEW

File: gfx/UltimateCataclysm/pngs_human_body_32x36/overlay/skin/skins_mutation.json [hunk 1]
```diff
@@ -41,12 +41,12 @@
   },
   {
     "id": "overlay_female_mutation_THICK_SCALES",
-    "fg": "scales_f",
+    "fg": "scales_thick_f",
```
Comment (Fris0uman) [inline gfx/UltimateCataclysm/pngs_human_body_32x36/overlay/skin/skins_mutation.json:44]: Same here, name the sprite with the correct id

File: gfx/UltimateCataclysm/pngs_human_body_32x36/overlay/mutations/tail_club/tail_club.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "id": "overlay_female_mutation_TAIL_CLUB",
```
Comment (Fris0uman) [inline gfx/UltimateCataclysm/pngs_human_body_32x36/overlay/mutations/tail_club/tail_club.json:3]: Why add this json? Just name the sprite with the correct id

File: gfx/UltimateCataclysm/pngs_human_body_32x36/overlay/skin/skins_mutation.json (hunk 1 above)
Comment (KittyTac) [reply gfx/UltimateCataclysm/pngs_human_body_32x36/overlay/skin/skins_mutation.json:44]: I copied how it was done in the older sprites. Gonna do that if that's the new standard.

**Comment by KittyTac** [comment]
@Fris0uman I did it.

**Pull request merged**